src/axom/core/Types.hpp

```diff
@@ -25,6 +25,22 @@
 
 namespace axom
 {
+/*
+  Axom integer types are deprecated.
+  Their use can trigger an warning or an error, or be quietly allowed,
+  by using cmake -DAXOM_DEPRECATED_TYPES=<WARN|ERROR|ALLOW>
```

Comment:
Minor: We've been consistently using `CMake` rather than `cmake`. Please update here and elsewhere.

Reply:
Done.